dist/index.html

```diff
@@ -12,19 +12,20 @@
   <body>
     <section class="home">
       <section class="home__header">
-        <div class="home__header2">
+        <div class="home__header__secondary"></div>
```

Comment:
Changing the class name here messes up the styling. Fix.

Reply:
i renamed it in css as well.

Reply:
refactoring in css for `.home__header2` effected the layouts  for home button and heading in a user page. I fixed the main page though. So i just renamed the div back to `<div class="home__header2">`.
Consulted with my mentor he is saying that
Having two tags with different class names, such as <div class="home__header"> and <div class="home__header2">, is not inherently a bad practice for accessibility. Class names are primarily used for styling and selecting elements with CSS and JavaScript, so from an accessibility perspective, it doesn't directly impact the accessibility of the content. in future was advised not to use `header2` in the naming. 
Instead  something like `<div class="home__header__secondary` should be used.